File: Iniciante/URI 2930.lua
line = {}
n = 0
for i in string.gmatch(io.read("*l"), "%S+") do
   n = n + 1
   line[n] = i
end
e = tonumber(line[1])
d = tonumber(line[2])

if (e > d) then
    print('Eu odeio a professora!')
elseif ((d - e) >= 3) then
    print('Muito bem! Apresenta antes do Natal!')
elseif ((e + 2) < 24) then
    print('Parece o trabalho do meu filho!\nTCC Apresentado!')
else
    print('Parece o trabalho do meu filho!\nFail! Entao eh nataaaaal!')
end

File: Iniciante/URI 2930.clj
(require '[clojure.string :as str])

(def line (str/split (read-line) #" "))
(def e (Integer/parseInt (get line 0)))
(def d (Integer/parseInt (get line 1)))

(cond
    (> e d) 
        (println "Eu odeio a professora!")
    
    (>= (- d e) 3) 
        (println "Muito bem! Apresenta antes do Natal!")
    
    (< (+ e 2) 24) 
        (println "Parece o trabalho do meu filho!\nTCC Apresentado!")
    
    :else 
        (println "Parece o trabalho do meu filho!\nFail! Entao eh nataaaaal!")   
)